system-parachains/constants/src/lib.rs

```diff
@@ -86,9 +86,13 @@ pub mod async_backing {
 	/// Operational  extrinsics.
 	pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(85);
 
+	/// Maximum PoV size.
+	// TODO: This value is copied from the recent update of the `polkadot_primitives::MAX_POV_SIZE`
```

Comment:
Not sure it's recorded in the style guide, but generally a TODO item in comments requires a corresponding Github issue

Reply:
done